backend/src/main/java/com/benine/backend/Main.java

```diff
@@ -64,6 +67,7 @@ public static void main(String[] args) {
       server.createContext("/move", new MovingHandler(cameraController));
       server.createContext("/zoom", new ZoomingHandler(cameraController));
       server.createContext("/preset", new PresetHandler(cameraController));
+      server.createContext("/public", new FileHandler(cameraController));
```

Comment:
Static would be a good name to since this endpoint seems to serve the static content.